proto/buf/alpha/registry/v1alpha1/doc.proto

```diff
@@ -184,8 +184,8 @@ message Service {
   Location location = 6;
   repeated Method methods = 7;
   ServiceOptions service_options = 8;
-  // file_deprecated is the deprecation status of the file containing the service.
-  bool file_deprecated = 9;
+  // implicitly_deprecated is true if its enclosing file or parent element is deprecated.
```

Comment:
Services too can't be enclosed in a parent element that I'm aware of so I'd tweak this comment to only mention enclosing file.